src/containers/LocationPage/LocationPage.jsx

```diff
@@ -333,37 +308,18 @@ class LocationPage extends PureComponent {
   }
 
   /**
-   * Callback for when a line is highlighted in time series
+   * Callback to show an incident and change the time aggregation to month
+   * @param {Array} ispIds Ids of ISPs to change
```

Comment:
I think the variable name "ispIds" should be removed from the docstring in line 312